_studio/shared/umc/codec/vp9_dec/include/umc_vp9_utils.h

```diff
@@ -31,6 +31,8 @@
 #define ALIGN_POWER_OF_TWO(value, n) \
     (((value) + ((1 << (n)) - 1)) & ~((1 << (n)) - 1))
 
+#define VP9_INVALID_REF_FRAME -1
```

Comment:
It's 2018, and C++ 11. Please move this into namespace and change define to constexpr auto.

Reply:
Consistency trumps fashion. Still, will fix it as you propose.